alignment/build.gradle

```
@@ -0,0 +1,56 @@
+plugins {
+    id 'maven-publish' // only used for publishing to local maven
+    id 'java-gradle-plugin' // applies "java" plugin and adds gradleApi() to dependencies
+}
+
+group 'org.jboss.pme.alignment'
```

I don't think using 'pme' is suitable here - or for that matter using 'alignment' as the task may do more than that ?
```suggestion
group 'org.jboss.gradle.plugin.build-analyser'
```

---

I was thinking of using `gme` for the top-level package but we certainly can go with your suggestion.

---

I settled on `gm` for gradle manipulator to keep the parallel with PME.

---

I still think the naming of ``alignment`` isn't necessarily appropriate for this stage - we're analysing the build and writing metadata hence thinking ``build-analyser`` ?